backend/src/mirrors_qa_backend/extract.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any
+from urllib.parse import urlsplit
+
+import pycountry
+import requests
+from bs4 import BeautifulSoup, NavigableString
+from bs4.element import Tag
+
+from mirrors_qa_backend import logger, schemas
+from mirrors_qa_backend.exceptions import MirrorsExtractError
+from mirrors_qa_backend.settings import Settings
+
+
+def get_current_mirrors() -> list[schemas.Mirror]:
+    """
+    Returns list of current mirrors from the mrirors url.
+
+    Raises MirrorsExtractError if the parser is unable to extract the mirrors
+    from the page. This is most likely as a result of the page being updated
+    indicating that the parsing logic should be updated
+    """
+
+    def find_country_rows(tag: Tag) -> bool:
+        """
+        Filters out table rows that do not contain mirror
```

Comment:
docstring convention is single line and if more than a single line, second line must be empty